templates/SilverStripe/Security/Security_MultiAuthenticatorTabbedForms.ss

```diff
@@ -2,13 +2,13 @@
     <ul class="nav nav-tabs" role="tablist">
         <% loop $Forms %>
         <li role="presentation" <% if Pos = 1 %>class="active"<% end_if %>>
-            <a href="#{$FormName}" aria-controls="{$FormName}" role="tab" data-toggle="tab">$AuthenticatorName</a>
+            <a href="#form_{$Pos}" aria-controls="form_{$Pos}" role="tab" data-toggle="tab">$AuthenticatorName</a>
```

Comment:
I swear a left a comment on this over the break but Github seems to have eaten it...

The identifier should continue to relate to this specific login form. Could we keep the generated FormName and append the Pos to it like so? `{$FormName}_{$Pos}`